--- /dev/fd/63	2026-02-26 11:49:56
+++ /dev/fd/62	2026-02-26 11:49:56
@@ -173,7 +173,7 @@
 
 
                              ---------- MATRIX TIMING ----------
-                               FILL: 1504 msec  FACTOR: 0 msec
+                               FILL: 1484 msec  FACTOR: 0 msec
 
 
                            -------- CURRENTS AND LOCATION --------
@@ -470,7 +470,7 @@
 
 
                              ---------- MATRIX TIMING ----------
-                               FILL: 616 msec  FACTOR: 0 msec
+                               FILL: 600 msec  FACTOR: 0 msec
 
 
                         --------- ANTENNA INPUT PARAMETERS ---------
